Tests/Tests.UWP/MainPage.xaml.cs

```diff
@@ -60,6 +60,8 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
                     }
 
                     _nunit.Options.ResultFilePath = Path.Combine(ApplicationData.Current.LocalFolder.Path, resultPath);
+                    var sfFileToCopy = await ApplicationData.Current.LocalFolder.GetFileAsync(resultPath);
+                    var sfOriginalFile = await sfFileToCopy.CopyAsync(ApplicationData.Current.LocalFolder, "nonConvertedTestResults.xml", NameCollisionOption.ReplaceExisting);
```

Comment:
Do we need this anymore?

Reply:
I actually forgot to fix this. This make the test fail because it should be on line 67. Considering that these 2 lines of code aren't only helpful for CI but also for the local creation and inspection of the test result, this could be handy for debugging the nito async issue in the stacktrace. Of course, they can be readded to a future branch that's specif to this nito issue.
Let me know if I should remove it.

Reply:
I don't think we need it - it's fairly easy to get to the xml by finding the app install directory.

Reply:
I'm a bit confused. That code isn't just to copy the file. Since the transformation replaces the original file, the code makes a copy of the original xml, so that we have both: the converted and the original.
But I'll remove it, since it's easy to re-add.